### PR TITLE
IAP-4780 - Updates to API Portal release notes Nov 21

### PR DESCRIPTION
**Pull request by dginevaAxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link
https://deploy-preview-2169--axway-open-docs.netlify.app/docs/apim_relnotes/20211130_apip_relnotes/

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by jowens-axway — CHANGES_REQUESTED** [review]
###Replacing Centos 8  as base OS
As Centos 8 is end of support in 2021, we are now providing API Portal in a docker container using the Alpine base OS instead. 

Also use "Elasticsearch" anywhere refer to Elastic, I believe that is the correct way to refer to to it.
Rest looks good!

**Comment by dginevaAxway** [comment]
I add the changes and add a reference for elastic search configuration section.

File: content/en/docs/apim_relnotes/20211130_apip_relnotes.md [hunk 1]
```diff
@@ -23,14 +23,17 @@ API Portal is available as a software installation or a virtualized deployment i
 
 The following new features and enhancements are available in this update.
 
-### Replacing Centos 8 as base OS
-As Centos 8 is end of support in 2021, we are now providing API Portal in a docker container using the Alpine base OS instead.
+### Centos 8 is replaced by Alpine
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20211130_apip_relnotes.md:26]: @dginevaAxway please have a look at my changes and let me know if it's ok to merge. Thanks

**Comment by dginevaAxway** [comment]
@andreamussap Everything looks ok except the base operational system, BaseOS is a technical term and make scene for the people I wont replace it with similar words.

**Comment by dginevaAxway** [comment]
@jowens-axway can you review this PR again for clarity if it's clear enough: (The publish/unpublish (enable/disable) toggle of Implicit flow will no longer affect)

**Comment by andreamussap** [comment]
> BaseOS

@dginevaAxway ok, I will change it back. Just waiting for @jowens-axway feedback in case there's something else to change. 
John, I was talking to Dimi that the 'OAuth client credentials flow improvements' section doesn't seem clear to me, the part of the 'Implicit flow' without linking to a 'see more in documentation'. What do you think?

**Comment by jowens-axway** [comment]
Am out of office but just a note on this - implicit flow being mentioned here is just complicating the message - we want to advertise that we now support client credentials flow inline with manager adding support.

I would tend to leave the message at that if possible.

**Pull request merged**